scompose/version.py

```diff
@@ -8,7 +8,7 @@
 
 """
 
-__version__ = "0.0.17"
+__version__ = "0.0.18"
 AUTHOR = "Vanessa Sochat"
```

Comment:
Great job doing this! We would want to add a note to the CHANGELOG.md in the root of the repository as well.